components/Container/Container.module.css

```diff
@@ -6,31 +6,40 @@
   align-items: center;
   justify-content: center;
   min-height: 250px;
-  width: 100%;
-  fill: currentColor;
+  fill: currentcolor;
+  flex-direction: column;
 }
 
-/* need to have a higher order of specificity since HeroBanner fights it */
-.Container.fullViewportHeight {
-  min-height: 100vh;
+@media screen and (--small-viewport) {
+  .Container > * {
+    width: 85%;
+    margin: 0 1.5rem;
+  }
 }
 
-.content {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  margin: 3.5rem auto;
-  width: 92.5%;
```

Comment:
Instead of all the changes you made to this file, I think you need only create a CSS variable for the nav bar's width and set the max-width to that value and make `width: calc(100% - 3.5rem);`

For the branding page specifically, the h6 just needs to be changed to a p and the 70ch max-width will take over.